app/stac_api/management/commands/remove_expired_items.py

```diff
@@ -0,0 +1,68 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management.base import CommandParser
+from django.utils import timezone
+
+from stac_api.models import Asset
+from stac_api.models import Item
+from stac_api.utils import CommandHandler
+from stac_api.utils import CustomBaseCommand
+
+
+class Handler(CommandHandler):
+
+    def delete(self, instance, object_type):
+        if self.options['dry_run']:
+            self.print_success(f'skipping deletion of {object_type} {instance}')
+        else:
+            instance.delete()
+
+    def run(self):
+        # print(self.options)
+        self.print_success('running command to remove expired items')
+        min_age_hours = self.options['min_age_hours']
+        self.print_warning(f"deleting all items expired longer than {min_age_hours} hours")
+        items = Item.objects.filter(
+            properties_expires__lte=timezone.now() - timedelta(hours=min_age_hours)
+        ).all()
+        for i in items:
+            assets = Asset.objects.filter(item_id=i.id).all()
```

Comment:
JFYI I think you could use reverse relations here: `i.assets.all()`

Reply:
Thanks. I changed it as I think it's nicer. :+1: